toolchain/check/subst.h

```diff
@@ -45,6 +45,11 @@ class SubstInstCallbacks {
     return orig_inst_id;
   }
 
+  // A helper for implementing `Rebuild` that constructs and returns an `InstId`
+  // for the `new_inst`.
```

Comment:
I find it's usually better to avoid starting a function comment by saying what the function is _for_, because it doesn't tell the reader anything about the contract. How about something like this?

```suggestion
  // Builds a new constant by evaluating `new_inst`, and returns its `InstId`.
```

Reply:
Done